esdm/esdm_drng_mgr.c

```diff
@@ -751,6 +753,8 @@ static ssize_t esdm_drng_get(struct esdm_drng *drng, uint8_t *outbuf,
 
 			/* Do not produce more than DRNG security strength. */
 			todo = min_uint32(todo, esdm_security_strength() >> 3);
+
+			atomic_set(&drng->internal_bits, 0);
```

Comment:
Can you please elaborate on this setting considerating the addition to esdm_drng_inject?

Reply:
I tried to do so in the latest commits.